content/documentation/rpc/node/get_log_stream.mdx

```diff
@@ -5,21 +5,39 @@ description: RPC Node | Iron Fish Documentation
 
 Streams logs from the node
 
-#### Request
+#### Request Body
 
 ```js
-undefined;
+undefined
 ```
 
 #### Response
 
 ```js
 {
-  level: string;
-  type: string;
-  tag: string;
-  args: string;
-  date: string;
+  level: string
+  type: string
+  tag: string
+  args: string
+  date: string
+}
+```
+
+### Example
+
+```bash
+# Request
+curl -X POST http://localhost:8021/node/getLogStream
+
+# Response
+{
+  "data": {
```

Comment:
Does this need a `status`?

Reply:
`status` isn't returned when using HTTP streaming